lib/diagnostic.js

```diff
@@ -6,23 +6,38 @@
 // splits it into an array of normalized words
 // (uppercase strings without punctuation)
 // and returns that array.
-const normalizeWords = function (/* */) {
-  /* your response here */
+const normalizeWords = function (paragraph) {
+  for (let i = 0; i < words.length; i++) {
+  let word = words[i].split(/\s+/).replace(/\W+/, '');
+  normalizedWords[i] = word.toUpperCase();
+  }
+  return paragraph;
 };
 
 // 2. Write a function that takes a string argument
 // and returns an array of unique normalized words.
-const uniqueWords = function (/* */) {
-  /* your response here */
+let paragraph = [];
+const uniqueWords = function (paragrapg) {
+  for ( let i = 0; i < paragraph.length; i++){
+    let word = normalizedWords[i];
+    normalizedWords = word.split();
+  }
+  return paragraph;
 };
```

Comment:
A few issues here:
1) make sure you spell your variables correctly
2) if `paragraph` is an empty array, what length will it have?
3) what is `normalizedWords`? is that term defined inside this function?